### PR TITLE
Sage 2.2.0 rc1

### PR DESCRIPTION
**Pull request by amcisaac**
This is release candidate 1 for the Sage 2.2.0 force field minor release. It has fixes for small ring internal angles and sulfonamide geometries. The details can be found here: https://github.com/openforcefield/sage-2.2.0/releases/tag/v2.2.0-rc1

### CODE REVIEW

File: README.md
Comment (mattwthompson) [inline README.md:168]: The versioning here is YYYY.MM.MINOR (We should add this to the README, I think it's copied over from another package.)

```suggestion
- `2024.03.0`: This release adds `openff-2.2.0-rc1.offxml` and `openff_unconstrained-2.2.0-rc1.offxml`, Sage 2.2.0 Release Candidate 1, which modifies some small ring internal angles and remedies issues with sulfamide geometries.
```

Is the plan to make a release (i.e. conda package) of this? I can coordinate this with @j-wags's signal

File: openforcefields/offxml/openff-2.2.0-rc1.offxml
Comment (j-wags) [inline openforcefields/offxml/openff-2.2.0-rc1.offxml:4]: Please update the date in both files

File: README.md
Comment (j-wags) [reply README.md:168]: Yes, the plan is to make a release+conda package with this. I'll do a quick review, then it'd be great if you coordinated the release.

**Review by j-wags — APPROVED** [review]
Approved pending two things:
1) updating the date
2) Uploading a "release tarball" somewhere (the most direct path would be adding a zipped version of the the entire folder where you cloned the sage-2.2.0 repo and ran the refit, with all the intermediate files and other goodies). This could either go in the release assets of the sage-2.2.0 release that you've already made, or the release assets of the openff-forcefield release that Matt will cut after merging this PR. I think you've made one before for 2.1.1, but an example of a release tarball with "all the goodies" of a QM fit is [here](https://github.com/openforcefield/openff-sage/releases/tag/2.0.0-rc.1)

File: .zenodo.json
Comment (j-wags) [inline .zenodo.json:11]:
```suggestion
      {
        "orcid": "0000-0002-6878-0192",
        "affiliation": "Open Force Field Consortium, Open Molecular Software Foundation, Davis,  CA, USA",
        "name": "Westbrook, Brent"
      },
```

**Comment by amcisaac** [comment]
> 2. Uploading a "release tarball" somewhere (the most direct path would be adding a zipped version of the the entire folder where you cloned the sage-2.2.0 repo and ran the refit, with all the intermediate files and other goodies). This could either go in the release assets of the sage-2.2.0 release that you've already made, or the release assets of the openff-forcefield release that Matt will cut after merging this PR. I think you've made one before for 2.1.1, but an example of a release tarball with "all the goodies" of a QM fit is [here](https://github.com/openforcefield/openff-sage/releases/tag/2.0.0-rc.1)

I thought I had done this by linking the release of the sage 2.2.0 repo, which has the tarball attached as “source code.tar.gz” (https://github.com/openforcefield/sage-2.2.0/releases/tag/v2.2.0-rc1). I just copied that format from the Parsley release that is linked in the new FF checklist. Is there a better place to upload it? I didn’t see the tarballs in any of the directories in this repo so wasn’t sure where to put it.

I’ll make all the other changes soon, thanks so much for your help!

File: README.md
Comment (amcisaac) [reply README.md:168]: Just to double check--is changing it from `2024.03.06` --> `2024.03.0` the same change as changing it to YYYY.MM.MINOR ? I can change it to `2024.03.MINOR` if that's preferable, just let me know.

**Comment by amcisaac** [comment]
Sounds good, I also updated the release notes to make it clear that all the code was included in the source code tarball, and I downloaded and unzipped it to make sure it's all there (it is!)

**Review by mattwthompson — APPROVED** [review]
Looks like all of Jeff's feedback is resolved, and everything looks good to me. I'll merge and cut the release shortly

**Comment by amcisaac** [comment]
Thank you both so much for all your help!

**Pull request merged**
